scripts/std-analysis.sh

```diff
@@ -104,7 +105,7 @@ for f in $results/*overall.csv; do
     fname=$(basename $f)
     crate=${fname%_scan_overall.csv}
     echo -n "$crate," >> $summary
-    tr -d [:alpha:]_,; < $f | tr -s '\n' ',' \
+    tr -d "[:alpha:]_,;" < $f | tr -s '\n' ',' \
```

Comment:
Thank you, but how did this ever work?

Reply:
I think it was a typo when I created the script from my local script. This script doesn't run as part of the CI, which is probably something that we need to address.

Reply:
Yes, would love for that to be in CI in some way or other!